common/djangoapps/course_modes/views.py

```diff
@@ -226,3 +228,31 @@ def _get_requested_mode(self, request_dict):
             return 'honor'
         else:
             return None
+
+
+@ensure_csrf_cookie
+@require_level('staff')
```

Comment:
Does this mean anyone with staff privileges can add new course modes? That might be too broad.  I'm pretty sure this is currently restricted to Marketing and Services to determine verified certificates and prices.